src/features/CreateGame.js

```diff
@@ -64,263 +37,101 @@ export const CreateGame = () => {
             console.log("error joining match in CreateGame clickHandler", err)
           )
           .then((player) => {
-            localStorage.setItem("matchID", matchTemp);
+            console.log(player);
             localStorage.setItem("name", name);
             localStorage.setItem("id", player.playerID);
             localStorage.setItem("credentials", player.playerCredentials);
-            // dispatch(setPlayerID(player.playerID));
-            // dispatch(setPlayerCredentials(player.playerCredentials));
           });
+        return match.matchID;
       })
-      .then(() => {
-        console.log("matchTemp", matchTemp);
-        navigate(`/waitingroom/${matchTemp}`);
+      .then((matchID) => {
+        console.log("matchTemp", matchID);
+        navigate(`/apples/:room/${matchID}`);
```

Comment:
instead of :room here, we can put waiting room,
so `/apples/:room/${matchID}` => `/apples/waitingroom/$(matchID}`   b/c we'll always be rerouting players to the waiting room after they create a game